doc/SCC-API-(Implemented).md

```diff
@@ -12,6 +12,73 @@ These several levels of authentication also act as scopes on the queried
 resources. Naturally, resource listings on one level will resemble subsets of
 the same listings on a higher level.
 
+## Differences in RMT, SMT and SCC API
+
+- The `available` attribute is only present in SMT/RMT and indicates if a certain
+  repository is mirrored or not.
+  *Affected endpoints*:
+    * `/connect/*/products`
+    * `/connect/systems/services`
+    * `/connect/systems/products/synchronize`
+    * `/connect/systems/activations`
+
+- The product `free` attribute is always true in RMT.
+  *Affected endpoints*:
+    * `/connect/*/products`
+    * `/connect/systems/services`
+    * `/connect/systems/products/migrations`
+    * `/connect/systems/products/synchronize`
+    * `/connect/systems/activations`
+
+- No `registration code` is required by RMT.
+  *Affected endpoints*:
+    * `/connect/subscriptions/products`
+
+- A `registration code` is not required by SMT. (If its supplied it will show up
+  in SCC if system information forwarding is enabled in SCC)
+  *Affected endpoints*:
+    * `/connect/subscriptions/products`
+
+- The `distro_target` attribute is only used by SMT.
```

Comment:
By "used" do you mean it consumes that attribute, or it adds it to the JSON responses? Maybe you could clarify.